GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kroxylicious Governance
+
+This document outlines the governance model for Kroxylicious.
+This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+[cfc]: https://www.commonhaus.org/bylaws/cf-council.html
+[cdm]: https://www.commonhaus.org/bylaws/decision-making.html
+[ctp]: https://www.commonhaus.org/policies/trademark-policy/
+[coc]: ./CODE_OF_CONDUCT.md
+[contrib]: ./CONTRIBUTING.md
+
+## Table of Contents
+
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+- [Trademark Policy](#trademark-policy)
+- [Contributing](#contributing)
+
+## Roles and Responsibilities
+
+- **Code owners:** Individuals with write access to the repository, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in each repository (`.github/CODEOWNERS`).
```

Comment:
In the codeowners branch https://github.com/kroxylicious/kroxylicious/pull/1886 it looks like we might refer to github teams. If we make those teams public would it be worth linking to them from here?

Reply:
@robobario that could get out of sync. Better to have a single path to a single truth imho.